Docker.md

```diff
@@ -61,9 +67,9 @@ Or a node with a set number of miners:
 docker-compose run --rm pegnet go run main.go --log=debug --miners=4
```

Comment:
`main.go` is now `pegnet.go` on develop. Please would you change this?

Reply:
done :)

Reply:
Sorry, my bad, line 60 also references main.go. Would you mind changing that too? Other than that problem, I have tested this and it works, so will merge as soon as that is changed.

Reply:
whoops, missed that one. Fixed @afenrir